src/lando/main/management/commands/start_uplift_worker.py

```diff
@@ -12,30 +11,98 @@
+
+[submit]
+auto_submit = True
+always_blocking = False
+warn_untracked = True
+
+[patch]
+apply_to = base
+create_bookmark = False
+create_topic = False
+create_branch = True
+always_full_stack = False
+branch_name_template = phab-D{rev_id}
+create_commit = True
+
+[updater]
+self_last_check = -1
+self_auto_update = False
+get_pre_releases = True
+arc_last_check =
+
+[error_reporting]
+report_to_sentry = True
+
+[telemetry]
+enabled = True
+""".lstrip()
+
 
 class Command(StartWorkerCommand):
     help = "Start an uplift worker."
     worker_class = UpliftWorker
 
     def setup_moz_phab_config(self):
-        """Copy the moz-phab config file to the user's home directory."""
-        source_config = settings.BASE_DIR / "api/legacy/workers/config/moz-phab-config"
+        """Write the moz-phab config file to the user's home directory."""
+        dest_config = Path.home() / ".moz-phab-config"
 
-        home_dir = Path.home()
-        dest_config = home_dir / ".moz-phab-config"
-
-        if not source_config.exists():
+        logger.info(f"Writing moz-phab config to {dest_config}.")
+        try:
+            dest_config.write_text(MOZ_PHAB_CONFIG_CONTENT, encoding="utf-8")
+        except OSError as exc:
             raise CommandError(
-                f"moz-phab config file not found at {source_config}. "
-                f"The uplift worker requires this configuration file "
-                f"to be loaded at {dest_config}."
-            )
+                f"Failed to write moz-phab config to {dest_config}: {exc}"
+            ) from exc
 
-        logger.info(f"Setting up moz-phab config: {source_config} -> {dest_config}.")
-        shutil.copy2(source_config, dest_config)
+        self.verify_moz_phab_config(dest_config)
         logger.info("moz-phab config installed successfully.")
 
     def handle(self, name: str, **options):
         """Set up moz-phab config before starting the worker."""
         self.setup_moz_phab_config()
         super().handle(name, **options)
+
+    def verify_moz_phab_config(self, dest_config: Path):
+        """Ensure moz-phab reads the generated configuration."""
+        try:
+            # Run `moz-phab --help`, which will print the location of the
+            # config file.
+            result = subprocess.run(
+                ["moz-phab", "--help"],
+                capture_output=True,
+                text=True,
+                check=True,
+            )
+        except FileNotFoundError as exc:
+            raise CommandError(
+                "moz-phab executable not found while verifying configuration."
+            ) from exc
+        except subprocess.CalledProcessError as exc:
+            raise CommandError(
+                "moz-phab configuration verification failed: " f"{exc}."
+            ) from exc
+
+        # Ensure the config file location we wrote our config file to
+        # is the destination in the `moz-phab --help` output.
+        if str(dest_config) not in result.stdout:
```

Comment:
nit: this could be a little more rigorous.

Reply:
Yeah, it's not ideal. I think to make any further improvements here we would need to add something to `moz-phab doctor`, or add a command like `moz-phab {config,debug}` etc.